vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts

```diff
@@ -38,13 +38,22 @@ export class PageViewPerformanceManager {
          *  |---network---||---request---|---response---|---dom---|
          *  |--------------------------total----------------------|
          */
+        var navigationTiming = this.getPerformanceNavigationTiming();
         var timing = this.getPerformanceTiming();
-        if (timing) {
-            var total = DateTimeUtils.GetDuration(timing.navigationStart, timing.loadEventEnd);
-            var network = DateTimeUtils.GetDuration(timing.navigationStart, timing.connectEnd);
-            var request = DateTimeUtils.GetDuration(timing.requestStart, timing.responseStart);
-            var response = DateTimeUtils.GetDuration(timing.responseStart, timing.responseEnd);
-            var dom = DateTimeUtils.GetDuration(timing.responseEnd, timing.loadEventEnd);
+        if (navigationTiming || timing) {
+            if (navigationTiming) {
```

Comment:
Please verify this is working correctly in IE and Safari

Reply:
Verified working correctly in IE